Dockerfile

```diff
@@ -18,6 +18,9 @@ COPY suggester/pom.xml /mvn/suggester/
 RUN sed -i 's:<module>distribution</module>::g' /mvn/pom.xml
 RUN sed -i 's:<module>tools</module>::g' /mvn/pom.xml
 
+# get clean state
+RUN mvn clean
```

Comment:
sometimes the docker image build barfs in tools build due to leftover `target/env` from local system build. This takes care of that.

Reply:
This does not help, does it? At this point we only copied `pom.xml` files to the docker image. I think we should modify `RUN mvn -DskipTests=true -Dmaven.javadoc.skip=true -B -V package` to include `clean` target as well, right?